src/content/contributing/funding-rounds/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Contributor funding rounds
+description: Information hub about funding rounds for ethereum.org contributors
+lang: en
+---
+
+# Contributor funding rounds {#contributor-funding-rounds}
+
+The [ethereum.org website team](/about/), with sponsorship from the [Ethereum Foundation](https://ethereum.foundation), is excited to be running a series of funding rounds to give back and reward community members who have made impactful contributions!
+
+<ButtonLink href="/contributing/funding-rounds/participating/" mt={8}>How to participate</ButtonLink>
+
+## What for? {#what-for}
+
+Ethereum.org is currently managed by a core team of ten full-time individuals, but is comprised of contributions from **thousands of contributors** from all over the globe!
+
+Contributions to ethereum.org have increased significantly in frequency over the past five years. While the vast majority of these contributions have been on a voluntary basis, large portions of the site content **would not have been possible** without this community support, and the core team has been investigating measures to return value to impactful contributors.
+
+<InfoBanner emoji="❤️" my="8">
+The goal is to retroactively reward impactful work, while encouraging and creating incentive for everyone to contribute to public goods, including ethereum.org. And of course, to say <strong>thank you</strong>.
+</InfoBanner>
+
+[Contributing to ethereum.org](/contributing/)
+
+## The challenge in deciding {#the-challenge-in-deciding}
+
+Deciding _which_ contributions are deserving of financial reward, and _how much_ they should receive, is _challenging_ and _subjective_.
+
+If a core team member finds a contribution valuable, this doesn't mean others did. Similarly, just because the core team did not appreciate the impact of a particular contribution, this doesn't mean that potentially hundreds of thousands of others didn't find immense value in the addition.
```

Comment:
```suggestion
If a core team member finds a contribution valuable, this doesn't mean others did. Similarly, just because the core team did not appreciate the impact of a particular contribution, this doesn't mean that potentially hundreds of thousands of others didn't find immense value in it.
```